src/main/java/uk/gov/hmcts/reform/wataskconfigurationapi/ccdmapping/variableextractors/ConstantVariableExtractor.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.wataskconfigurationapi.ccdmapping.variableextractors;
+
+import uk.gov.hmcts.reform.wataskconfigurationapi.thirdparty.camunda.CamundaValue;
+import uk.gov.hmcts.reform.wataskconfigurationapi.thirdparty.camunda.TaskResponse;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@SuppressWarnings({"PMD.UseConcurrentHashMap"})
+public class ConstantVariableExtractor implements TaskVariableExtractor {
+    public static final String STATUS_VARIABLE_KEY = "taskState";
+
+    @Override
+    public Map<String, Object> getValues(TaskResponse task, Map<String, CamundaValue<Object>> processVariables) {
+        Map<String, Object> mappedDetails = new HashMap<>();
+        mappedDetails.put(STATUS_VARIABLE_KEY, "configured");
```

Comment:
Perhaps the status should also be enums, just thinking because it would be easier to copy to other services for the functional tests and stuff.